test/Proposals.ts

```diff
@@ -0,0 +1,43 @@
+import {ethers, upgrades} from "hardhat";
+import {UpgradableCounterContract} from "../typechain-types";
+import {expect} from "chai";
+
+
+describe("Proposals test", function () {
+    it("SoftwareUpgradeProposal", async function () {
+        const originalImplProxy: UpgradableCounterContract = await upgrades.deployProxy(
+            await ethers.getContractFactory('UpgradableCounterContract'),
+            [],
+            {kind: 'uups'}
+        );
```

Comment:
Rename `originalImplProxy` to `upgradableProxy`. You should use only a single proxy and replace the implementation in it.